source/guides/network/maintaining-a-symbol-node.rst

```diff
@@ -0,0 +1,125 @@
+.. post:: 06 Jun, 2021
+    :category: Network
+    :excerpt: 1
+    :nocomments:
+
+#########################
+Maintaining a Symbol Node
+#########################
+
+Once you have created a |codename| node, be it :doc:`manually <running-a-symbol-node-manually>` or :doc:`using Symbol bootstrap <running-a-symbol-node>`, the node is **mostly autonomous**. But there is still a little bit of maintenance required, explained in this guide.
+
+- Regular nodes only need to take care of the :ref:`maintain-server-version` section.
+
+- :ref:`Voting nodes <voting-node-program>` also need to read the :ref:`maintain-voting-keys` section.
+
+.. _maintain-server-version:
+
+Server updates
+**************
+
+New |codename| server versions will be released **periodically**. If you want to benefit from the added features and bugfixes you will need to **update your node** to use the latest server version.
+
+Additionally, if you have enrolled in any of the :doc:`../../concepts/reward-programs` keeping up with server versions is a **mandatory requirement**.
```

Comment:
Do we need to mention the reward program at all since the program has been postponed?

Reply:
Well, the Reward Programs page has been published for a while now and I am not going to take it down. When the program is activated having the nodes up-to-date will surely be a requirement so I like this sentence here :)

Reply:
After talking with @klminehan-nem I've removed the mention of the reward programs.